app/views/settings/email/_email_fieldset.html.erb

```diff
@@ -18,13 +18,25 @@
       <% end %>
     </div>
 
-  <%= f.govuk_email_field :service_email_output,
-    label: {
-      text: f.object.class.human_attribute_name(:service_email_output),
-      for: settings_field_id(f, :service_email_output, deployment_environment)
-    },
-    width: 'two-thirds',
-    id: settings_field_id(f, :service_email_output, deployment_environment) %>
+<div data-component="Expander">
+    <div class="govuk-form-group govuk-!-margin-left-1 <%= f.object.errors.present? ? 'govuk-form-group--error' : '' %>">
+      <% if f.object.errors.present? %>
+        <% f.object.errors.full_messages.each do |error_message| %>
+            <p id="email-settings-base-<%= deployment_environment %>-field-error" class="govuk-error-message">
+              <span class="govuk-visually-hidden">Error:</span> <%= error_message %>
+            </p>
+        <% end %>
+      <% end %>
+      <%= f.govuk_email_field :service_email_output,
+        label: {
+          text: f.object.class.human_attribute_name(:service_email_output),
+          for: settings_field_id(f, :service_email_output, deployment_environment)
+        },
+        hint:{ text: t('activemodel.attributes.email_settings.service_email_output_hint') },
+        width: 'two-thirds',
+        id: settings_field_id(f, :service_email_output, deployment_environment) %>
```

Comment:
OK, so this is a little complicated.  But because this block is used twice in the page for diffrent envs we need the `id`s of things to distinguish between them.  Because this uses the GOVUK formbuilder gem, its generating an id for the hint for us, using the field key (`:service_email_output`) which produces an `id` for the hint of 'email-settings-service-email-output-hint' which is resulting in duplicate `id`s on the page.

To resolve this I think, we need to use a proc for the hint, as when we do that we are responsible for generating all the html and an `id` isn't autogenerated.
See the docs: https://govuk-form-builder.netlify.app/introduction/labels-captions-hints-and-legends/#input-erb-setting-hint-content-with-a-proc
and the source code for the hint: https://github.com/DFE-Digital/govuk-formbuilder/blob/main/lib/govuk_design_system_formbuilder/elements/hint.rb

```suggestion
        width: 'two-thirds',
        id: settings_field_id(f, :service_email_output, deployment_environment),
        hint: -> do %> 
        <div class="govuk-hint" 
             id="email-settings-service-email-output-<%=deployment_environment%>-hint">
             <%=t('activemodel.attributes.email_settings.service_email_output_hint') %>
        </div>
      <% end%>
```

Reply:
Nice thanks Chris, this seems to have worked:

### Before
<img width="1136" alt="Screenshot 2023-01-27 at 15 18 30" src="https://user-images.githubusercontent.com/29227502/215121598-590e50ba-640f-412e-b7b2-ade69f6697e0.png">

### After:
<img width="1187" alt="Screenshot 2023-01-27 at 15 17 26" src="https://user-images.githubusercontent.com/29227502/215121652-e52d9033-ff80-488d-9167-6206d5154bbb.png">

<img width="1245" alt="Screenshot 2023-01-27 at 15 16 51" src="https://user-images.githubusercontent.com/29227502/215122103-0d2bc333-ddcb-4d94-97b6-4f4370423bd3.png">